modules/virt-importing-vm-wizard.adoc

```diff
@@ -2,24 +2,37 @@
 //
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+
 [id='virt-importing-vm-wizard_{context}']
-ifdef::virt-importing-vmware-vm[]
-= Importing a VMware virtual machine or template with the virtual machine wizard
+= Importing a VM with the VM Import wizard
 
-You can import a VMware virtual machine or template by using the virtual machine wizard.
-endif::[]
-ifdef::virt-importing-rhv-vm[]
-= Importing a Red Hat Virtualization virtual machine with the virtual machine wizard
+You can import a single VM with the VM Import wizard.
 
-You can import a Red Hat Virtualization (RHV) virtual machine by using the virtual machine wizard.
+ifdef::virt-importing-vmware-vm[]
+You can also import a VM template. If you import a VM template, {VirtProductName} creates a virtual machine based on the template.
+
+[discrete]
+== Prerequisites
```

Comment:
This should be `.Prerequisites`.

Reply:
OK, I misunderstood the new guidelines. Didn't realize we're supposed to use '[discrete]' in assemblies and '.Header' in modules.

Reply:
@apinnick Just to be clear, you're not supposed to use [discrete] when doing == Prerequisites in assemblies. You're supposed to let it show in the TOC.

Reply:
Understood.